LayoutTests/fast/forms/color/color-input-keyboard-focus.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../../resources/js-test.js"></script>
+</head>
+<body>
+
+<input type="color" id="color">
+
+<script>
+    description("Make sure that keyboard navigation doesn't skip the input with type color.");
+
+    if (window.eventSender) {
+        eventSender.keyDown("\t");
```

Comment:
`keyDown` events are not supported by 'iOS' test infrastructure. So you have to add 'platform-specific' expectation for 'iOS' in 'TestExpectation' file in `LayoutTests\platforms\ios` to skip this test.

Reply:
@pxlcoder - I think 'safe-merge-queue' will fail because of this on 'iOS', should I remove label from this PR?

Reply:
Yes, good catch, I've removed the label.

Reply:
Rather than skip on iOS, we should rewrite so it works everywhere. `UIHelper.keyDown` is the way to go.

Example here: https://github.com/WebKit/WebKit/blob/main/LayoutTests/fast/forms/textfield-outline.html#L11

Note that you'll need to make the test async now, similar to that example.

Reply:
ok

Reply:
Needs some investigation as to why it failed.

Reply:
@pxlcoder anything more I could/should do here?

Reply:
Have you done any debugging of the test on iOS (simulator)? That would be the next step.

Reply:
Oops no, I will try to do that the coming days